### PR TITLE
Pods Refining

### PR DESCRIPTION
**Pull request by WilliamWeberBerrutti**
This PR attempts to fix https://github.com/OpenHV/OpenHV/issues/885 and fix https://github.com/OpenHV/OpenHV/issues/877 and contemplates this discussion https://github.com/OpenHV/OpenHV/discussions/879.

### CODE REVIEW

File: mods/hv/rules/pods.yaml [hunk 1]
```diff
@@ -212,15 +198,18 @@ TECHNICIAN:
...
-		Queue: Pod
-		Description: Financial analyst.\nInvests into stock market for dividents.\n  Unarmed
-		Prerequisites: ~module.yi, techcenter
-		BuildPaletteOrder: 60
+	# Disabled for now, for skirmish and multiplayer games
+	# Buildable:
+	# 	Queue: Pod
+	# 	Description: Financial analyst.\nInvests into stock market for dividents.\n  Unarmed
+	# 	Prerequisites: techcenter
+	# 	BuildPaletteOrder: 6
```
Comment (Mailaender) [inline mods/hv/rules/pods.yaml:212]:
```suggestion
	 Buildable:
	 	Queue: Pod
	 	Description: Financial analyst.\nInvests into stock market for dividents.\n  Unarmed
	 	Prerequisites: ~disabled
	 	BuildPaletteOrder: 6
```

Comment (Mailaender) [reply mods/hv/rules/pods.yaml:212]: Also, don't mix tabs and spaces.

File: mods/hv/rules/buildings.yaml [hunk 2]
```diff
@@ -1035,7 +1035,7 @@ ORESMELT:
 	Buildable:
 		Queue: Building
 		Description: Ore Smelter.\nProcesses resources for extra credits.
-		Prerequisites: ~structures.sc, techcenter
+		Prerequisites: techcenter
```
Comment (Dzierzan) [inline mods/hv/rules/buildings.yaml:1038]: Keep it for SC for now. I'll provide new utility vehicle which will replace Broker Pod.

File: mods/hv/rules/pods.yaml [hunk 3]
```diff
@@ -212,15 +198,18 @@ TECHNICIAN:
...
 
 BROKERPOD:
 	Inherits: ^Pod
 	Inherits@Selection: ^SelectableSupportUnit
-	Buildable:
-		Queue: Pod
-		Description: Financial analyst.\nInvests into stock market for dividents.\n  Unarmed
-		Prerequisites: ~module.yi, techcenter
-		BuildPaletteOrder: 60
+	# Disabled for now, for skirmish and multiplayer games
```
Comment (Dzierzan) [inline mods/hv/rules/pods.yaml:207]: Revert those changes. I'll provide new utility vehicle which will replace Broker Pod.

File: mods/hv/rules/buildings.yaml (hunk 2 above)
Comment (Dzierzan) [inline mods/hv/rules/buildings.yaml:1038]: Keep it for SC for now. I'll provide new utility vehicle which will replace Broker Pod.

File: mods/hv/rules/pods.yaml [hunk 4]
```diff
@@ -344,18 +333,16 @@ BOMBERPOD:
...
 ELECTRICPOD:
 	Inherits: ^Pod
-	Inherits@AutoTarget: ^AutoTargetGroundAssaultMove
+	Inherits@AutoTarget: ^AutoTargetAllAssaultMove
 	Buildable:
 		Queue: Pod
-		Description: Breakthrough vehicle.\nArmed with lightning gun.\n  Strong vs Pods, Vehicles and Buildings.
-		Prerequisites: ~module.yi, techcenter
-		BuildPaletteOrder: 60
+		Description: Fast scout vehicle.\nShoots surface to air/ground electricity.\n  Strong vs Tanks and Aircraft\n  Weak vs Pods
```
Comment (Dzierzan) [inline mods/hv/rules/pods.yaml:339]: Well, naming it a scout is too much IMO. I mean it's too big to be the scout. Maybe `Support vehicle`?

**Review by Dzierzan — CHANGES_REQUESTED** [review]

File: mods/hv/weapons/energetic.yaml [hunk 5]
```diff
@@ -183,13 +183,33 @@ VoltageArc:
...
 	Inherits: VoltageArc
-	Range: 6c0
+	Range: 5c512
 	Report: lighting_gun_freqman.wav
+	ValidTargets: Ground, Water, Air, Tree, Lava, Swamp
+	ReloadDelay: 60
 	Projectile: EnergyBolt
 		Radius: 2
+		Arm: 2
+		Inaccuracy: 0
```
Comment (Dzierzan) [inline mods/hv/weapons/energetic.yaml:193]: `EnergyBolt` doesn't have parameters `Arm` and `Inaccuracy`. Remove those two, please.

**Review by Dzierzan — CHANGES_REQUESTED** [review]

File: mods/hv/rules/pods.yaml (hunk 3 above)
Comment (WilliamWeberBerrutti) [reply mods/hv/rules/pods.yaml:207]: As we discussed in the Matrix channel, if I understood correctly, for now, I'll allow both factions to build Ore Smelters.

File: mods/hv/rules/buildings.yaml (hunk 2 above)
Comment (WilliamWeberBerrutti) [reply mods/hv/rules/buildings.yaml:1038]: As we discussed in the Matrix channel, if I understood correctly, for now, I'll allow both factions to build Ore Smelters.

**Comment by Dzierzan** [comment]
Maybe it would be better to squash all commits but have one detailed description. @Mailaender your thoughts about this?

**Comment by Mailaender** [comment]
You don't have to squash this. I can do it during merge, but single commits make it more readable.

File: mods/hv/rules/pods.yaml [hunk 6]
```diff
@@ -173,7 +159,7 @@ TECHNICIAN:
 	Inherits@Selection: ^SelectableSupportUnit
 	Buildable:
 		Queue: Pod
-		Description: Field technician.\nInfiltrates and captures enemy structures.\n  Unarmed
+		Description: Field technician.\nInfiltrates and captures enemy structures.\nHeals pods.\n  Unarmed
```
Comment (Mailaender) [inline mods/hv/rules/pods.yaml:162]:
```suggestion
		Description: Field technician.\nInfiltrates and captures enemy structures.\nRepairs pods.\n  Unarmed
```

**Pull request merged**